### PR TITLE
meta-monitoring: add tracing template

### PR DESCRIPTION
**Pull request by frzifus**
```bash
$ k  get pods -n observatorium-traces-testing
NAME                                                             READY   STATUS    RESTARTS   AGE
tempo-observatorium-tempostack-compactor-7664f4bdc6-7wlj7        1/1     Running   0          2m36s
tempo-observatorium-tempostack-distributor-7bc6fc87b7-q9h5t      1/1     Running   0          2m36s
tempo-observatorium-tempostack-gateway-659ccfff95-9mvt8          2/2     Running   0          2m36s
tempo-observatorium-tempostack-ingester-0                        1/1     Running   0          2m36s
tempo-observatorium-tempostack-querier-64ff859788-lhb8s          1/1     Running   0          2m36s
tempo-observatorium-tempostack-query-frontend-7ff987bc67-mhbkc   2/2     Running   0          2m36s
```

### CODE REVIEW

File: resources/services/meta-monitoring/tracing-template.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  This template deploys Tempo via a TempoStack CR.
+parameters:
+  - name: NAMESPACE
+    description: The namespace where Tempo will be installed.
+    required: true
+    value: observatorium-tools
+  - name: TENANT_NAME
+    value: application
+  - name: TENANT_ID
+    value: 42ffe36a-c2c9-4cfc-b876-72ca5036aca7
```
Comment (philipgough) [inline resources/services/meta-monitoring/tracing-template.yaml:17]: where does this value come from?

Comment (frzifus) [reply resources/services/meta-monitoring/tracing-template.yaml:17]: It was generated by me. (used it as a default value) - Should it be removed?

Comment (philipgough) [reply resources/services/meta-monitoring/tracing-template.yaml:17]: No it is more of a general question I guess, some default is fine. So how are tenants configured with this operator?

Comment (frzifus) [reply resources/services/meta-monitoring/tracing-template.yaml:17]: In openshift mode, we have to define a tenant. Next we need to setup a service account and a role to define the tenant permission and link it to a specific application. This application needs to report the `X-Scope-OrgID` and `Bearer-Token` to write/read data. ([Tempo authentication sample](https://github.com/os-observability/redhat-rhosdt-samples/blob/7d6b8a5a79376bea09ba740be3adb8bda018f384/tempo/auth/README.md))

Comment (philipgough) [reply resources/services/meta-monitoring/tracing-template.yaml:17]: I don't think this is being used anywhere. It is fine for now to hardcode the tenant as you have done down below

File: resources/services/meta-monitoring/tracing-template.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      template:
+        gateway:
+          enabled: true
+        queryFrontend:
+          jaegerQuery:
+            enabled: true
+  - apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: rhobs-tempo-traces-reader
```
Comment (philipgough) [inline resources/services/meta-monitoring/tracing-template.yaml:63]: should this be read-writer ?

**Review by philipgough — APPROVED** [review]
lgtm

**Pull request merged**